modules/core/include/opencv2/core/hal/intrin_sse.hpp

```diff
@@ -2987,6 +3004,11 @@ inline v_float32x4 v_cvt_f32(const v_int32x4& a)
     return v_float32x4(_mm_cvtepi32_ps(a.val));
 }
 
+static inline v_float32x4 v_cvt_f32(const v_uint32x4& a)
```

Comment:
the same question about `static`

Reply:
Becase I want to make compiler to do forced inline this function to speed-up it's work.